.github/workflows/scanner-offline-bundle-update.yaml

```diff
@@ -3,9 +3,6 @@ on:
   schedule:
     - cron: '0 */3 * * *'
   workflow_dispatch:
-  push:
-    branches:
-    - scanner/*
```

Comment:
Checking: I remember @daynewlee added this to allow triggering this on development branches. Is it now obsolete?

Reply:
I believe so. Can you confirm @daynewlee ?

Keeping this here can also be dangerous if someone were to make a branch with this prefix